generate/generate.go

```diff
@@ -21,7 +21,8 @@ var (
 
 // Generator represents a generator for a container spec.
 type Generator struct {
-	spec *rspec.Spec
+	spec         *rspec.Spec
+	hostSpecific bool
```

Comment:
Why make this private with a setter (and no getter)?  Is there a reason to not make it public?